BB-SC/MWLayer/Communication/BB_Communication_Service_S2S.md

```diff
@@ -0,0 +1,76 @@
+
+# Communication Service (S2S)
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Communication
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+
+## Known Implementation
+
+## ID (unique name)
+
+## Description
+<!-- General Description of the BB -->
+Define standard service for secured offboard communication
+
```

Comment:
Unclear what this BB is supposed to do? Which problem is solved, how?

Reply:
Agreed. Needs further described. -> WP2

Reply:
rename to Communiction_Server_S2S and fill with content from Word file (p. 64)

Reply:
add new field: "Expert-discussion", status: "in discussion"

Reply:
done